website/components/homepage-hero/HomepageHero.module.css

```diff
@@ -14,11 +14,26 @@
     background: none;
     padding: 0;
 
-    & :global(.g-btn) {
+    & :global(.g-btn:nth-of-type(1)) {
       border-color: var(--boundary);
       background-color: var(--boundary);
     }
 
+    & :global(.g-btn:nth-of-type(2)) {
+      color: var(--gray-1);
+
+      &::after {
+        display: inline-block;
+        margin-left: 5px;
+        vertical-align: middle;
+        content: ' ';
+        background-image: url('/img/icon-right-arrow.svg');
+        background-size: 20px 20px;
+        height: 20px;
+        width: 20px;
+      }
+    }
+
```

Comment:
This is a bit of a miserable hack to style the secondary hero CTA button to the design spec. Our `Hero` component doesn't allow for direct variant theming.

Reply:
Should we maybe put in a ticket to address this one? I hate to just leave hacks out like this haha